zombiereloaded/ze_cyberspace_v1.cfg

```diff
@@ -0,0 +1 @@
+mp_freezetime 5
```

Comment:
Server already defaults to 5 second freezetime, if map is overriding this then you need to fix that (it will override map config too).